### PR TITLE
remove parallel-test

### PR DESCRIPTION
**Pull request by nsinkov**
## Changes proposed in this PR

- remove parallel-test

## Why are we making these changes?

parallel-test hangs, replace with eftest

### CODE REVIEW

**Comment by nsinkov** [comment]
@shamsimam updated

File: waiter/project.clj [hunk 1]
```diff
@@ -111,11 +111,13 @@
                  [slingshot "0.12.2"]
                  [try-let "1.3.1"
                   :exclusions [org.clojure/clojure]]]
-  :eftest {:report clojure.test/report}
+  :eftest {:report clojure.test/report
+           :thread-count (fn []
+                           (or (some-> (System/getenv "LEIN_TEST_THREADS") Long/valueOf)
```
Comment (shamsimam) [inline waiter/project.clj]: `thread-count` eventually calls `Executors/newFixedThreadPool` which expect an int argument.

**Comment by nsinkov** [comment]
updated

File: waiter/project.clj [hunk 2]
```diff
@@ -111,11 +111,12 @@
                  [slingshot "0.12.2"]
                  [try-let "1.3.1"
                   :exclusions [org.clojure/clojure]]]
-  :eftest {:report clojure.test/report}
+  :eftest {:report clojure.test/report
+           :thread-count (or (some-> (System/getenv "LEIN_TEST_THREADS") Integer/valueOf)
```
Comment (shamsimam) [inline waiter/project.clj]: Should we do `parseInt` instead of `valueOf`?
`availableProcessors` also returns an `int`.

File: waiter/project.clj [hunk 3]
```diff
@@ -111,11 +111,12 @@
...
+                           (.availableProcessors (Runtime/getRuntime)))}
   :resource-paths ["resources"]
   :main waiter.main
-  :plugins [[com.holychao/parallel-test "0.3.2"]
-            [lein-exec "0.3.7"]
+  :plugins [[lein-exec "0.3.7"]
             [test2junit "1.2.2"]
             [lein-eftest "0.5.8"]]
```
Comment (shamsimam) [inline waiter/project.clj]: Nitpick: lexicographic order

**Pull request merged**